data/app.js

```diff
@@ -0,0 +1,15 @@
+const Dirwatcher = require('./dirwatcher.js');
+const Importer = require('./importer.js');
+
+const dirwatcher = new Dirwatcher();
+const importer = new Importer();
+
+const dirPath = '/Users/Yahor_Pamakha/js-workspace/data/';
+
+dirwatcher.on('changed', (filename) => {
```

Comment:
Consider exercising "separation of concerns" here, making both "dirwatcher" and "importer" and their respective listeners unaware of each other.
In other words, use a separate new EventEmmiter that itself listens to a change event and fires importer logic.